File: src/main/java/org/jukeboxmc/inventory/BrewingStandInventory.java
package org.jukeboxmc.inventory;

import com.nukkitx.protocol.bedrock.data.inventory.ContainerType;
import org.jetbrains.annotations.NotNull;
import org.jukeboxmc.blockentity.BlockEntityBrewingStand;

/**
 * @author LucGamesYT
 * @version 1.0
 */
public class BrewingStandInventory extends ContainerInventory {

    public BrewingStandInventory( InventoryHolder holder) {
        super( holder, -1, 5 );
    }

    @Override
    public BlockEntityBrewingStand getInventoryHolder() {
        return (BlockEntityBrewingStand) this.holder;
    }

    @Override
    public @NotNull InventoryType getType() {
        return InventoryType.BREWING_STAND;
    }

    @Override
    public @NotNull ContainerType getWindowTypeId() {
        return ContainerType.BREWING_STAND;
    }
}

File: src/main/java/org/jukeboxmc/inventory/BrewingStandInventory.kt
package org.jukeboxmc.inventory

import com.nukkitx.protocol.bedrock.data.inventory.ContainerType
import org.jukeboxmc.blockentity.BlockEntityBrewingStand

/**
 * @author LucGamesYT
 * @version 1.0
 */
class BrewingStandInventory(holder: InventoryHolder?) : ContainerInventory(holder, -1, 5) {
    override val inventoryHolder: InventoryHolder?
        get() = holder as BlockEntityBrewingStand
    override val type: InventoryType
        get() = InventoryType.BREWING_STAND
    override val windowTypeId: ContainerType
        get() = ContainerType.BREWING_STAND
}